light/odr.go

```diff
@@ -119,6 +119,23 @@ func (req *BlockRequest) StoreResult(db ethdb.Database) {
 	rawdb.WriteBodyRLP(db, req.Hash, req.Number, req.Rlp)
 }
 
+type hashOrNumber struct {
+	Hash   common.Hash
+	Number uint64
+}
+
+type HeaderRequest struct {
+	OdrRequest
+	Origin hashOrNumber
+	Header *types.Header
+}
+
+func (req *HeaderRequest) StoreResult(db ethdb.Database) {
+	rawdb.WriteHeader(db, req.Header)
+	rawdb.WriteTd(db, req.Header.Hash(), req.Header.Number.Uint64(), big.NewInt(int64(req.Header.Number.Uint64()+1)))
```

Comment:
Is there a function somewhere that will calculate the TD for our consensus engine?  Seems that we're hard coding `Number+1` all over the place for TD.

Reply:
Not that I know of, usually `WriteTd` uses `Block.TotalDifficulty()` which just does this same thing `Number+1`